.vscode/launch.json

```diff
@@ -6,6 +6,9 @@
       "name": "Run Jest",
       "type": "node",
       "program": "${workspaceFolder}/node_modules/jest/bin/jest",
+      // These environment variables need to match what `yarn test` uses,
+      // see the definition of the "test" script in package.json
+      "env": { "LC_ALL": "C", "TZ": "UTC" },
```

Comment:
would it be possible and easier to tell vscode to run `yarn test` instead?

Reply:
Yes it is, using `runtimeExecutable` + `runtimeArgs`. I've changed it to use that.